.dockerignore

```diff
@@ -0,0 +1,58 @@
+.vscode/*
+!.vscode/settings.json
```

Comment:
Why keeping `.vscode/settings.json` in a docker ignore.
Maybe ignoring the whole `.vscode` would be better